cmd/mt-whisper-importer-reader/main.go

```diff
@@ -109,11 +109,12 @@ func main() {
 	fileChan := make(chan string)
 
 	wg := &sync.WaitGroup{}
+	wg.Add(*threads)
```

Comment:
oh right. I first thought i can get rid of the waitgroups and created a commit, then i realized that doesn't work and modified it with the `wg` but forgot to update the msg.